mmv1/products/compute/terraform.yaml

```diff
@@ -1020,6 +1031,8 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       serviceLabel: !ruby/object:Overrides::Terraform::PropertyOverride
         validation: !ruby/object:Provider::Terraform::Validation
           function: 'validateGCEName'
+      ipVersion: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_value: :IPV4
```

Comment:
```suggestion
```

This would need to be removed at the same time as the block in mmv1/products/compute/api.yaml